src/applications/gi/components/profile/BenefitsForm.jsx

```diff
@@ -14,11 +15,13 @@ export class BenefitsForm extends React.Component {
     eligibilityChange: PropTypes.func,
     showHeader: PropTypes.bool,
     handleInputFocus: PropTypes.func,
+    giBillChapterOpen: PropTypes.arrayOf(PropTypes.bool),
```

Comment:
use of this isn't currently needed since the only child currently uses one of the checks already present where this is used.

Added more for clarity and in case the conditions for a conditionally displayed question change